data/json/items/ammo.json

```diff
@@ -19,6 +19,18 @@
     "//2": "1 battery is 1 kJ of energy. 40 batteries is an alkaline C-cell.",
     "fuel": { "energy": 1 }
   },
+  {
+    "type": "AMMO",
+    "id": "rock",
+    "copy-from": "rock",
+    "name": "rock",
+    "description": "An ordinary rock",
```

Comment:
```suggestion
    "description": "An ordinary rock.",
```

Reply:
change made. finished making recipe and rebalancing staff sling. Also made changes to ammo weight and volume.